### PR TITLE
add .toString() and .valueOf() codebyte examples

### PR DESCRIPTION
**Pull request by andrzejkukuryk**
### Description

Added codebyte examples to date .toString() and .valueOf() methods.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@andrzejkukuryk This looks good except for one thing: Can we change the variable names in the examples so they don't refer to `currentDate`? We want to do this because the date used won't be current for anyone who reads this entry.

**Comment by andrzejkukuryk** [comment]
> @andrzejkukuryk This looks good except for one thing: Can we change the variable names in the examples so they don't refer to `currentDate`? We want to do this because the date used won't be current for anyone who reads this entry.

It's done! Current name is "exampleDate".

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@andrzejkukuryk Looks good for a second review.

File: content/javascript/concepts/dates/terms/toString/toString.md [hunk 1]
```diff
@@ -39,3 +39,15 @@ console.log(currentDateString);
 // Output: Thu Jul 15 2021 11:48:57 GMT-0700 (Pacific Daylight Time)
 // Note: Timezone may vary
 ```
+
+## Codebyte Example
+
+The following example shows the work of `.toString()` method.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/toString/toString.md:45]:
```suggestion
The following is runnable, and demonstrates the use of the `.toString()` method.
```

File: content/javascript/concepts/dates/terms/valueOf/valueOf.md [hunk 2]
```diff
@@ -34,3 +34,12 @@ console.log(date);
 console.log(date.valueOf());
 // Output: 915177600000
 ```
+## Codebyte Example
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/valueOf/valueOf.md:37]: It appears that this entry already has a codebyte and was incorrectly added to this pr. Please revert these changes.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@andrzejkukuryk I've added a couple of comments. Please update the entry, and tag me when you've finished. Thanks

**Comment by andrzejkukuryk** [comment]
@caupolicandiaz @SSwiniarski I guess it's ready to review.

**Review by caupolicandiaz — APPROVED** [review]
@andrzejkukuryk Looks good

**Review by SSwiniarski — APPROVED** [review]

**Pull request merged**